docker-compose.test.yaml

```diff
@@ -51,6 +51,11 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
 
+  mysql-test:
```

Comment:
call it sql-injection-test-mysql (and create a postgres container only for this test so that we keep separation)

Reply:
Done.

Reply:
Done.